src/db/migrations/structure.sql

```diff
@@ -408,6 +408,7 @@ CREATE TABLE public."Units" (
     "movementSpeed" integer NOT NULL,
     "createdAt" timestamp with time zone NOT NULL,
     "updatedAt" timestamp with time zone NOT NULL,
+    "pointsCost" integer DEFAULT 10 NOT NULL,
```

Comment:
I must not have done `npm run dump_db` last time after adding this field, so it's added here now.